automation/utilities/platform_utils.py

```diff
@@ -131,6 +133,9 @@ def get_configuration_string(manager_params, browser_params, versions):
     config_str += '\n\n'
     config_str += tabulate(table_input, headers=key_dict)
 
+    config_str += "\n\n========== JS Instrument Settings ==========\n"
+    config_str += json.dumps(js_config, indent=2, separators=(',', ': '))
```

Comment:
Seeing as the JS config is already a json string we're now getting escaped JSON printed.